www/docs/blog/posts/2023-10-08-slsa-generation-for-your-artifacts.md

```diff
@@ -0,0 +1,235 @@
+---
+date: 2023-10-08
```

Comment:
just need to update the date here and in the filename, and IMHO we can merge this :D